src/prob/pf.jl

```diff
@@ -692,7 +692,7 @@ function _cpf_load_interface(load::Dict{String,<:Any}, v_start::Dict{<:Any,<:Any
             y = g+im*b
             if wires == 3
                 y_prim = diagm(y)
-            elseif wires == 4
+            elseif wires == 4 || wires == 2
```

Comment:
this looks like something that should eventually be improved based on a flag track whether the data is Kron reduced or not, but it's probably fine for now - if we throw errors/warnings when we know the data is Kron reduced. Let's do a quick call